packages/website/components/Sidebar.tsx

```diff
@@ -84,66 +82,69 @@ const components: Array<SidebarSectionType | SidebarLinkType> = [
-          <SidebarLink
-            isActive={isLinkActive('/getting-started', currentPage)}
-            href="/getting-started"
-          >
-            Getting started
-          </SidebarLink>
-          <SidebarLink
-            isActive={isLinkActive('/contributing', currentPage)}
-            href="/contributing"
-          >
-            Contributing to Forma 36
-          </SidebarLink>
-          <Box marginBottom="spacingL" />
-          <SidebarSection
-            title="Forma 36 version 3"
-            links={sidebarLinks.forma36Version3}
-            currentPage={currentPage}
-          />
+    <Flex
+      as="nav"
+      aria-label="Main Navigation"
+      flexDirection="column"
+      className={styles.nav}
+    >
+      <List className={styles.list}>
+        {currentPage.includes('guidelines') && (
           <SidebarSection
             title="Guidelines"
             links={sidebarLinks.guidelines}
             currentPage={currentPage}
           />
+        )}
+        {currentPage.includes('tokens') && (
           <SidebarSection
             title="Tokens"
             links={sidebarLinks.tokens}
             currentPage={currentPage}
           />
-          <SidebarSection
-            title="Components"
-            links={components}
-            currentPage={currentPage}
-          />
-          <SidebarSection
-            title="Advanced components"
-            links={sidebarLinks.advancedComponents || []}
-            currentPage={currentPage}
-          />
-          <SidebarSection
-            title="Utils"
-            links={sidebarLinks.utils}
-            currentPage={currentPage}
-          />
-          <SidebarSection
-            title="Integrations"
-            links={sidebarLinks.integrations}
-            currentPage={currentPage}
-          />
-        </List>
-      </Flex>
-    </>
+        )}
+        {currentPage.includes('components') && (
+          <>
+            <SidebarSection
+              title="Components"
+              links={components}
+              currentPage={currentPage}
+            />
+            <SidebarSection
+              title="Utils"
+              links={sidebarLinks.utils}
+              currentPage={currentPage}
+            />
+            <SidebarSection
+              title="Integrations"
+              links={sidebarLinks.integrations}
+              currentPage={currentPage}
+            />
+          </>
+        )}
+        {currentPage.includes('') && (
+          <>
+            <SidebarLink
+              isActive={isLinkActive('/getting-started', currentPage)}
+              href="/getting-started"
+            >
+              Getting started
+            </SidebarLink>
+            <SidebarLink
+              isActive={isLinkActive('/contributing', currentPage)}
+              href="/contributing"
+            >
+              Contributing to Forma 36
+            </SidebarLink>
+            <Box marginBottom="spacingL" />
```

Comment:
Ah I almost forget

Could you remove all these `Box` used as spacing elements?

It's better to use CSS for this so we do not create empty HTML tags in the DOM

Reply:
sure, I wanted to tacke it in the next PR. Side bar is not styles properly yet.

Reply:
Ah ok! that makes sense